decidim-core/app/views/decidim/participatory_processes/show.html.erb

```diff
@@ -18,3 +18,5 @@
     </div>
   </div>
 <% end %>
+
+<%= comments_for participatory_process %>
```

Comment:
Do you think we could add comments in the `pages` feature instead? It's already on master and makes way more sense to me.

Otherwise it's a weird feature to have in `master`.